config/ftbquests/quests/chapters/tools.snbt

```diff
@@ -3079,62 +3101,100 @@
 			y: -16.0d
 			subtitle: "Iron jetpacks have only 1 iron jetpack?!? Ironic."
 			description: [
-				"Second tier of jetpacks are made of Bronze, Silver and wait for it, Iron! (Finally) Of course you need any jetpack from Tier 1 to craft Tier 2."
+				"Second tier of jetpacks introduce &ofinally&r Iron jetpack!"

```

Comment:
`Second tier of jetpacks introduce finally Iron jetpack!` Is grammatically incorrect. Not sure what you were going for here tbh - `Second tier of jetpacks finally introduce the Iron Jetpack`?

Reply:
Yea... I don't fully like those iron jetpack quests >.> But I honestly think the old ones were worse.
Gonna fix it up 👍